docs/software/connect.mdx

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I think this file is in the wrong place, should be moved to the end of the 01 light section